apps/components/src/app/menu/app.ts

```diff
@@ -0,0 +1,45 @@
+import { Component } from '@angular/core';
+import { NgpButton } from 'ng-primitives/button';
+import { NgpMenuTrigger } from 'ng-primitives/menu';
+import { Menu } from './menu';
+
+@Component({
+  selector: 'app-menu-example',
+  imports: [Menu, NgpButton, NgpMenuTrigger],
+  template: `
+    <button [ngpMenuTrigger]="menu" ngpButton>Open Menu</button>
+
+    <ng-template #menu>
+      <app-menu />
```

Comment:
I think we would need to make it so that menu items can be defined as content rather than within the Menu component, because menu items will be different for every menu in the app. We should probably create a separate MenuItem component:

```html
<ng-template #menu>
      <app-menu>
              <button app-menu-item>Item 1</button>
              <button app-menu-item>Item 2</button>
              <button app-menu-item>Item 3</button>
      </app-menu>
</ng-template>